internal/services/pubkey_service.go

```diff
@@ -99,6 +109,50 @@ func DeletePubkey(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	pubkey, err := dao.GetPubkeyDao(r.Context())
```

Comment:
I believe this is not necessary, we have th `pubkeyDao` already and it is the same dao now (after the merge of daos).
We can fetch the pubkey by ID here (we should also render 404 if the pubkey is not found).
```suggestion
	pubkey, err := pubkeyDao.GetByID(r.Context(), id)
```